configs/sot/siamese_rpn/README.md

```diff
@@ -14,12 +14,26 @@
 }
 ```
 
-## Results and models on LaSOT dataset
+## Results and models
 
-We observe around 1.0 points fluctuations in Success and 1.5 points fluctuations in Norm percision. We provide the best model.
+### LaSOT
 
-Note that all of checkpoints from 11-th to 20-th epoch need to be evaluated in order to achieve the best results.
+Note that the checkpoints from 10-th to 20-th epoch will be evaluated during training. You can find the best checkpoint from the log file.
+
+We observe around 1.0 points fluctuations in Success and 1.5 points fluctuations in Norm percision. We provide the best model with its configuration and training log.
 
 |    Backbone     |  Style  | Lr schd | Mem (GB) | Inf time (fps) | Success | Norm precision | Config | Download |
 | :-------------: | :-----: | :-----: | :------: | :------------: | :----: | :----: | :------: | :--------: |
 |    R-50    |  -  |   20e    | 7.54        | 50.0              | 49.9 | 57.9 | [config](siamese_rpn_r50_1x_lasot.py) | [model](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_lasot/siamese_rpn_r50_1x_lasot_20201218_051019-3c522eff.pth) &#124; [log](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_lasot/siamese_rpn_r50_1x_lasot_20201218_051019.log.json) |
+
+### UAV123
+
+First, you can train the model following [quick_run](https://github.com/open-mmlab/mmtracking/blob/master/docs/quick_run.md#training). Note that the checkpoints from 10-th to 20-th epoch will be evaluated during training and the best checkpoint can be found from the log file.
+
+Then, you need to use the best checkpoint to search the hyperparameters in UAV123 following [here](https://github.com/open-mmlab/mmtracking/blob/master/docs/useful_tools_scripts.md#siameserpn-test-time-parameter-search) to achieve the best results.
```

Comment:
We can merge Line 31-33 to
`The checkpoints from 10-th to 20-th epoch will be evaluated during training`

`After training, you need to pick up the best checkpoint from the log file, then use the best checkpoint to search the hyperparameters on UAV123 following [here](https://github.com/open-mmlab/mmtracking/blob/master/docs/useful_tools_scripts.md#siameserpn-test-time-parameter-search) to achieve the best results.`